tools/src/Packages.ts

```diff
@@ -168,10 +168,10 @@ export class Package {
       );
     } else if (platform === 'android') {
       // On Android we need to read expoview's build.gradle file
-      const buildGradle = fs.readFileSync(path.join(ANDROID_DIR, 'expoview/build.gradle'), 'utf8');
+      const buildGradle = fs.readFileSync(path.join(ANDROID_DIR, 'settings.gradle'), 'utf8');
```

Comment:
```suggestion
      const settingsGradle = fs.readFileSync(path.join(ANDROID_DIR, 'settings.gradle'), 'utf8');
```
And also update the comment above 😄

Reply:
Nice catch, thanks 👍